openpype/hosts/houdini/plugins/create/create_pointcache.py

```diff
@@ -13,8 +13,22 @@ class CreatePointCache(plugin.HoudiniCreator):
     icon = "gears"
 
     def create(self, subset_name, instance_data, pre_create_data):
+
+        # Allow creating the
```

Comment:
```suggestion
        # Allow creating the ...
```
missing completing this